airflow/example_dags/example_subdag_operator.py

```diff
@@ -0,0 +1,55 @@
+from datetime import datetime
+
+from airflow.models import DAG
+from airflow.operators import DummyOperator, SubDagOperator
+
+from airflow.example_dags.subdags import subdag
+
+
+args = {
+    'owner': 'airflow',
+    'start_date': datetime(2016, 1, 1),
+}
+
+dag = DAG(
+    dag_id='example_subdag_operator',
+    default_args=args,
+    schedule_interval="@daily",
```

Comment:
Change the schedule_interval to `@once` to avoid a lot of scheduling work when examples are loaded. For example, in 1 year, a newbie may not want his/her system churning out 365 dag_runs for this example. The other option is to set the start_date to a few days in the past, but I'd prefer the `@once` because I don't think the start_date can be reset for a DAG once it is loaded the first time. Hence, if an example is reloaded later, the scheduler will be mightily busy.